MiniApp/Classes/Extensions/URLSession+SessionProtocol.swift

```diff
@@ -0,0 +1,14 @@
+extension URLSession: SessionProtocol {
+    func startDataTask(with request: URLRequest, completionHandler: @escaping (Result<ResponseData, Error>) -> Void) {
```

Comment:
I'd prefer to split this into multiple lines too, if possible.

Reply:
Done